health/mapping/es.py

```diff
@@ -76,11 +76,34 @@
 
 
 def init_elastic(es, index_to_create="ms_health_idx_1"):
-    r = requests.get("%s/%s" % (es, "ms_health"))
-    if r.status_code == 200:
-        print("Index ms_health already exists, nothing to do!")
+    # TODO(kzaitsev): replace prints with logging
+    r = requests.get("%s/%s" % (es, index_to_create))
+    if r.ok:
+        print("Index {} already exists, nothing to do!".format(
```

Comment:
looks like we can just issue the put request multiple times without penalty, right?